models/holes/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub fn model(radius: impl Into<Scalar>, core: &mut fj::core::Core) -> Solid {
     let radius = radius.into();
 
     let size = radius * 4.;
-    let cuboid = cuboid::model([size * 2., size, size], core);
+    let cuboid = ::cuboid::model([size * 2., size, size], core);
```

Comment:
Why this change? It doesn't seem to make a difference, except to add visual noise.